exercises/concept/tim-from-marketing/.meta/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "authors": [
+    "smcg468"
+  ],
+  "files": {
+    "solution": [
+      "src/main/java/Badge.java"
+    ],
+    "test": [
+      "src/test/java/BadgeTests.java"
```

Comment:
Nitpicking here, but following the convention used by the other exercises this should be `BadgeTest.java`. So singular, not plural.

Could you rename the file and change it in the `config.json` here?